From fbb415a6ad15195146ef1f2720032ba17555d38c Mon Sep 17 00:00:00 2001
From: Su Baocheng <baocheng.su@siemens.com>
Date: Wed, 20 Jan 2021 16:52:19 +0800
Subject: [PATCH 27/28] iot2050: Add support for flash protect

IOT2050 use a Winbond flash that support lock & unlock, but it was not
enabled.

This winbond flash is a similar one to STMICRO, and can reuse the lock
unlock logic for STMICRO chips. So by adding CONFIG_SPI_FLASH_STMICRO
to iot2050_defconfig, the flash protection could be supported.

Signed-off-by: Su Baocheng <baocheng.su@siemens.com>
---
 configs/iot2050_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/iot2050_defconfig b/configs/iot2050_defconfig
index 38619f3f7b..1439434cdf 100644
--- a/configs/iot2050_defconfig
+++ b/configs/iot2050_defconfig
@@ -96,6 +96,7 @@ CONFIG_MMC_SDHCI_ADMA=y
 CONFIG_MMC_SDHCI_AM654=y
 CONFIG_DM_SPI_FLASH=y
 CONFIG_SPI_FLASH_SFDP_SUPPORT=y
+CONFIG_SPI_FLASH_STMICRO=y
 CONFIG_SPI_FLASH_WINBOND=y
 # CONFIG_SPI_FLASH_USE_4K_SECTORS is not set
 CONFIG_PHY_TI=y
-- 
2.25.1

